### PR TITLE
Excpetion handler

### PR DESCRIPTION
**Pull request by VNTee**
Global handler for exceptions like 400s and 500s

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/error.jsp [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <spring:url value="/resources/css/materialize.css" var="css" />
+    <spring:url value="/resources/js/materialize.js" var="js" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link type="text/css" rel="stylesheet" href="${css}"  media="screen,projection"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Error</title>
+</head>
+<body>
+<div class="container">
+    ERROR
```
Comment (imbirevkotlya) [inline src/main/webapp/WEB-INF/jsp/error.jsp]: can you create some good static page for errors?

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/com/epam/handler/GlobalExceptionHandlerController.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@ControllerAdvice
+@EnableWebMvc
+public class GlobalExceptionHandlerController {
+
+    private static final String DEFAULT_ERROR_VIEW = "error";
+
+    @ExceptionHandler(Exception.class)
```
Comment (imbirevkotlya) [inline src/main/java/com/epam/handler/GlobalExceptionHandlerController.java]: Please fix this with Throwable at this moment

File: src/main/java/com/epam/handler/GlobalExceptionHandlerController.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package com.epam.handler;
+
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@ControllerAdvice
+@EnableWebMvc
```
Comment (imbirevkotlya) [inline src/main/java/com/epam/handler/GlobalExceptionHandlerController.java]: Why we need @EnbleWebMvc annotation here?

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/com/epam/handler/GlobalExceptionHandlerController.java (hunk 3 above)
Comment (VNTee) [reply src/main/java/com/epam/handler/GlobalExceptionHandlerController.java]: Yes, you right, this annotation is superfluous

File: src/main/webapp/WEB-INF/jsp/error.jsp (hunk 1 above)
Comment (VNTee) [reply src/main/webapp/WEB-INF/jsp/error.jsp]: Done

File: src/main/java/com/epam/handler/GlobalExceptionHandlerController.java (hunk 2 above)
Comment (VNTee) [reply src/main/java/com/epam/handler/GlobalExceptionHandlerController.java]: Done

**Review by goponenkoev — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by MaltsevaKsenia — APPROVED** [review]

**Review by grigiv — APPROVED** [review]

**Pull request merged**